### ISSUE TITLE
Is Semantic-ui-vue2 ready for Server side rendering?

### ISSUE THREAD

**Issue by ansarizafar** [almino/semantic-ui-vue2#4]
Is it possible to use Semantic-ui-vue2 with NUXTJs?

**Comment by almino** [comment]
I'm not sure. Never tried Nuxt.js.

This is my first project ever using Vue.js. I'm still learning about what I can do with it.

Semantic UI for Vue2 (semantic-ui-vue2) has no dependency. It runs Vanilla JS. I can't see an caveat with Nuxt.js.

**Comment by ansarizafar** [comment]
I will try Semantic-ui-vue2 with Nuxt.js next week. I have used Semantic-ui css with Vuejs in another project [restock.gitlab.io](restock.gitlab.io)  Semantic UI and Vuejs are great. Thanks for working on this project. Is there any plan for adding more semantic-ui modules like accordion and sidebar?

**Comment by almino** [comment]
Yes. Accordion was the main reason I started this project. But I needed modal and dropdown first. I'm working on multiple selection dropdown.

**Comment by askhat** [comment]
Hey guys,

I'm considering using `semantic-ui-vue2` with `nuxt.js` in the next project. Did anyone encounter an srr-related, or any other, issues?

**Comment by pkej** [comment]
Not using NPM, at least. First error message is that it looks for something in ".src/.....". So it is missing some type of environment variable in includes, or something
